Trappist/src/Promact.Trappist.Repository/Account/AccountRepository.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Threading.Tasks;
+using Promact.Trappist.DomainModel.ApplicationClasses.Account;
+using Microsoft.AspNetCore.Identity;
+using Promact.Trappist.Web.Models;
+using Promact.Trappist.DomainModel.DbContext;
+
+namespace Promact.Trappist.Repository.Account
+{
+    public class AccountRepository : IAccountRepository
+    {
+        private readonly TrappistDbContext _dbcontext;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Remove the '_dbcontext' field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://server12:9005/coding_rules#rule_key=csharpsquid%3AS1450)